server/src/resolvers/UserResolver.ts

```diff
@@ -29,4 +29,13 @@ export class UserResolver {
 			};
 		}
 	}
+	@Mutation(() => UserResponse, {
+        description: "Updates user.",
+    })
+    async updateUser(
+        @Arg("data", () => UpdateUserType) data: UpdateUserType
+    ) {
```

Comment:
also add validation for the different stuff that is updated, see utils/validation.ts for the functions that are implemented